setup.py

```diff
@@ -142,7 +142,7 @@
         'SLIST = obspy.io.ascii.core',
         'PICKLE = obspy.core.stream',
         'CSS = obspy.io.css.core',
-        'DATAMARK = obspy.io.datamark.core',
+        'WIN = obspy.io.win.core',
```

Comment:
@ThomasLecocq do you want to add a deprecation or leave as is? We plan to freeze 1.1.0 on Sunday so if this PR should be in there it should be ready by then.